src/main/webapp/app/views/discovery/discovery-context.html

```diff
@@ -1,11 +1,34 @@
 <div class="discovery-context row">
+  {{$window}}
   <div class="dc-heading">
     <h2>{{discoveryContext.name}}</h2>
   </div>
   <div class="dc-content">
     <breadcrumbs contexts="breadcrumbContexts" reload="resetPage" home=""></breadcrumbs>
     <div class="col-md-3 dc-side-panel">
-      <div ng-style="{'background-image':'url({{discoveryContext.logoUrl}})'}" ng-if="discoveryContext.logoUrl" class="panel panel-default information-box logo-information-box">
+      <div ng-if="discoveryContext.logoUrl && discoveryContext.wideLogoUrl">
+        <div ng-style="{'background-image':'url({{discoveryContext.logoUrl}})'}" class="panel panel-default information-box logo-information-box narrow-logo">
```

Comment:
According to [this](https://docs.angularjs.org/api/ng/directive/ngStyle) there are known issues with using string interpolation within the ngStyle. This whole object should be expressed in the controller on the scope. We can use ES6 style interpolation at that point. BUT if we do that we will also need to add `"esversion": 6` to the .jshintrc (as I did [here](https://github.com/TAMULib/SAGE/pull/318/files#diff-c3ca886e85c3a97b5cc4c34d04e5b9d5c78359b1e69fd09b74a842b5dea81bb0L1))

Reply:
In addition to the interpolation issue, it would also be preferable to keep all that json out of the view, imo.